### PR TITLE
Survey Form (PR)

### PR DESCRIPTION
**Pull request by Nelofarzabi**
In this Project:

I created an empty repo.

- Copied code from (https://github.com/ZuhRa-HashiMi/Review_The_projects) which was completed during the Microverse Trials

- Created a new branch and added the copied code.

- Opened a pull request

- Played the role of Code Reviewer:

- by checking to see if [semantic HTML tags] were used.

- Checking to see if [HTML & CSS best practices] were applied.

- Added comments after reviewing the project as per the HTML Semantics tags, and HTML and CSS best practices

### CODE REVIEW

**Comment by Nelofarzabi** [comment]
README file is not updated. Always remember to update your README file in all your projects. I would highly suggest you update the template and add all the necessary information about this project.

**Comment by Nelofarzabi** [comment]
Kindly Submit commits regularly to keep track of changes made while you code. Commit messages can adequately communicate why a change was made, and understanding that makes development and collaboration more efficient. Read this article to have a better understanding https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/

File: README.md [hunk 1]
```diff
@@ -1 +1,288 @@
-# Project_Review
```
Comment (Nelofarzabi) [inline README.md:1]: README file is not updated. Always remember to update your README file in all your projects. It is important to fill your README file with information describing your project. I would highly suggest you add all the necessary information about this project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
+<!Doctype html>
```
Comment (Nelofarzabi) [inline index.html:1]: Kindly Submit commits regularly to keep track of changes made while you code. Commit messages can adequately communicate why a change was made, and understanding that makes development and collaboration more efficient. Read this article to have a better understanding https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/

**Review by Nelofarzabi — COMMENTED** [review]
Hi @Nelofarzabi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights 💪
✔️ Descriptive pull request
✔️ the project is working well on my pc
A very good attempt replicating the original survey website.
Required Changes ♻️
Check the comments below under review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @teke85 in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Nelofarzabi** [comment]
Kindly Submit commits regularly to keep track of changes made while you code. Commit messages can adequately communicate why a change was made, and understanding that makes development and collaboration more efficient. Read this article to have a better understanding https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/ 😄 😄

**Comment by Nelofarzabi** [comment]
README file is not updated. Always remember to update your README file in all your projects. It is important to fill your README file with information describing your project. I would highly suggest you add all the necessary information about this project. 😄 😄

**Comment by Nelofarzabi** [comment]
Hi @Nelofarzabi As per the project requirements, you were supposed to add an attribution note and acknowledgement in your README file. It's always important to leave attribution or acknowledge when you use somebody else's project. Kindly remember to do it. 👍🏻 👍🏻

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Nelofarzabi , 

# Requires Changes 🍀
You have done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 🫡

## Highlights
- Great summary ✅

## Required Changes ♻️

- Great job using githubs `review changes` feature✨. However, please also add at least 2 inline comments to your review as it is one of the project requirements🙏

image

You can use this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) as a reminder for how to use inline comments🫡

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+                name="user-recommend"
+                value="not-sure"
+                type="radio"
+                class="input-radio"
+              />Not sure</label
+            >
+          </div>
+      
+          <div class="form-group">
+            <p>
```
Comment (Nelofarzabi) [inline index.html:98]: Hi @Nelofarzabi Kindly use an h3 or h4 tag instead of a p tag. h4 tags are used for subheadings.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form From Trials</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
```
Comment (Nelofarzabi) [inline index.html:11]: Hi @Nelofarzabi Great work so far, kindly wrap the div tag inside a section tag.  Because section tag are semantic tag. 👍🏻

**Comment by Nelofarzabi** [comment]
Hi @Shisui6 Thank you for the observation. I have added inline comments. 👍🏻

**Review by huseyinbiyik — APPROVED** [review]
Hi @Nelofarzabi,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- You used the inline comments feature of Github. 👍 
- You used polite language instead of orders 👍 
- You mentioned why you are requesting these changes 👍 


## Optional suggestions
- You can use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**